tests/Feature/NumberTest.php

```diff
@@ -0,0 +1,179 @@
+<?php
+
+use FOP\Number\Number;
+
+test('format', function () {
+    $this->assertSame('0', Number::format(0));
+    $this->assertSame('1', Number::format(1));
+    $this->assertSame('10', Number::format(10));
+    $this->assertSame('25', Number::format(25));
+    $this->assertSame('100', Number::format(100));
+    $this->assertSame('100,000', Number::format(100000));
+    $this->assertSame('100,000.00', Number::format(100000, precision: 2));
+    $this->assertSame('100,000.12', Number::format(100000.123, precision: 2));
+    $this->assertSame('100,000.123', Number::format(100000.1234, maxPrecision: 3));
+    $this->assertSame('100,000.124', Number::format(100000.1236, maxPrecision: 3));
+    $this->assertSame('123,456,789', Number::format(123456789));
+
+    $this->assertSame('-1', Number::format(-1));
+    $this->assertSame('-10', Number::format(-10));
+    $this->assertSame('-25', Number::format(-25));
+
+    $this->assertSame('0.2', Number::format(0.2));
+    $this->assertSame('0.20', Number::format(0.2, precision: 2));
+    $this->assertSame('0.123', Number::format(0.1234, maxPrecision: 3));
+    $this->assertSame('1.23', Number::format(1.23));
+    $this->assertSame('-1.23', Number::format(-1.23));
+    $this->assertSame('123.456', Number::format(123.456));
+
+    $this->assertSame('∞', Number::format(INF));
+    $this->assertSame('NaN', Number::format(NAN));
+});
+
+test('format with different locale', function () {
+    $this->assertSame('123,456,789', Number::format(123456789, locale: 'en'));
+    $this->assertSame('123.456.789', Number::format(123456789, locale: 'de'));
+    $this->assertSame('123 456 789', Number::format(123456789, locale: 'fr'));
+    $this->assertSame('123 456 789', Number::format(123456789, locale: 'ru'));
+    $this->assertSame('123 456 789', Number::format(123456789, locale: 'sv'));
+});
+
+test('format with app locale', function () {
+    $this->assertSame('123,456,789', Number::format(123456789));
+
+    Number::useLocale('de');
+
+    $this->assertSame('123.456.789', Number::format(123456789));
+
+    Number::useLocale('en');
+});
+
+test('spellout', function () {
+    $this->assertSame('ten', Number::spell(10));
+    $this->assertSame('one point two', Number::spell(1.2));
+});
+
+test('spellout with locale', function () {
+    $this->assertSame('trois', Number::spell(3, 'fr'));
+});
+
+test('ordinal', function () {
+    $this->assertSame('1st', Number::ordinal(1));
+    $this->assertSame('2nd', Number::ordinal(2));
+    $this->assertSame('3rd', Number::ordinal(3));
```

Comment:
Missing test for `th` suffix:

```suggestion
    $this->assertSame('3rd', Number::ordinal(3));
    $this->assertSame('4rd', Number::ordinal(4));
```